src/frame_graph/frame_graph.hpp

```diff
@@ -220,6 +220,9 @@ namespace wr
 
 		inline void Resize(RenderSystem & render_system, std::uint32_t width, std::uint32_t height)
 		{
+			std::uint32_t m_width_scalar = (width * 100000u)  / m_old_window_size_width;
```

Comment:
This scaler should be moved to resizerendertarget instead and be a part of the properties struct.